numba/tests/test_array_methods.py

```diff
@@ -1805,6 +1805,55 @@ def test_array_ctor_with_dtype_arg(self):
         args = n, np.dtype('f4')
         np.testing.assert_array_equal(pyfunc(*args), cfunc(*args))
 
+    def test_frombuffer_offset(self):
```

Comment:
A good way to test these functions is to copy the NumPy tests and compile them with Numba. Not all tests will pass but this is a good way to find any corner case in the current implementation.

Reply:
I can't find np.frombuffer specific tests from the numpy repo

Reply:
I couldn't find it either. That's fine! I'll review the PR again some time this week.